scripts/upgrade_firmware.py

```diff
@@ -228,22 +228,22 @@ def read_num(self, bytes):
     def send(self):
         self.is_sent = True
         init_bytes = [chr(0x1), self.cmd, 
-                      chr((len(self.out) / 001) % 256),
-                      chr((len(self.out) / 256) % 256)]
+                      chr((len(self.out) / 0o001) % 256),
+                      chr((len(self.out) / 0o256) % 256)]
```

Comment:
I don't think these numbers should actually be octal. (1 is going to the same in either base, and the 256 didn't have a leading 0 in the original).